lib/core/utils.js

```diff
@@ -386,26 +388,13 @@ RESUtils.pageType = function() {
 	}
 
 	var currURL = location.href.split('#')[0];
-	var pageType = '';
-	if (RESUtils.regexes.wiki.test(currURL)) {
-		pageType = 'wiki';
-	} else if (RESUtils.regexes.search.test(currURL)) {
-		pageType = 'search';
-	} else if (RESUtils.regexes.profile.test(currURL)) {
-		pageType = 'profile';
-	} else if (RESUtils.regexes.comments.test(currURL)) {
-		pageType = 'comments';
-	} else if (RESUtils.regexes.inbox.test(currURL)) {
-		pageType = 'inbox';
-	} else if (RESUtils.regexes.submit.test(currURL)) {
-		pageType = 'submit';
-	} else if (RESUtils.regexes.prefs.test(currURL)) {
-		pageType = 'prefs';
-	} else if (RESUtils.regexes.stylesheet.test(currURL)) {
-		pageType = 'stylesheet';
-	} else {
-		pageType = 'linklist';
-	}
+	var defaultPageType = 'linklist';
+	var pageTypes = [ 'wiki', 'search', 'profile', 'comments', 'inbox', 'submit', 'prefs', 'stylesheet' ];
+	var unsortedPageTypes = Object.getOwnPropertyNames(RESUtils.regexes).filter(function(pageType) { return pageType !== 'all' && pageTypes.indexOf(pageType) === -1; }).reverse();
```

Comment:
This breaks everything because most pages that used to be `linklist` match `subredditPostListing`